webpack.config.js

```diff
@@ -111,6 +111,14 @@ const webpackConfig = {
 				exclude: /node_modules[\/\\](?!notifications-panel)/,
 				loader: [ 'happypack/loader' ],
 			},
+			{
+				test: /node_modules[\/\\](redux-form|react-redux)[\/\\]es/,
+				loader: 'babel-loader',
```

Comment:
you could change this to:
```js
  test: ....,
  use: [
    'cache-loader',
    {
       loader: 'babel-loader',
       options: { 
         babelrc: false,
         plugins: ...
       } 
     }
  ]
```

Reply:
We would have to install `cache-loader` as a new dependency, but maybe Happypack provides similar services?

Reply:
maybe we just move off happy-pack and onto cache-loader / thread-loader?

Reply:
I don't think we would need to! cache-loader comes bundled with webpack AFAIK (could be wrong).

regardless, this shouldn't be a slow transform by any means so I don't even know if we'd benefit from caching. Feel free to ignore this whole line of questioning :)

> maybe we just move off happy-pack and onto cache-loader / thread-loader?

🤷‍♂️

Reply:
When I tried to add the `use: 'cache-loader'` snipped from your comment above, it failed because the loader wasn't found. And there's a [NPM package](https://www.npmjs.com/package/cache-loader) for it.

Anyway, definitely a topic for another PR.